Automattic/wp-calypso
# Framework: Remove `lib/user` event emitter and unused types

#### Changes proposed in this Pull Request

This PR removes the `lib/user` event emitter from Calypso 🎉 

Closes #24004 where we took some time to reduxify `lib/user`.

#### Testing instructions

* Go to `/me` and verify saving your profile still works as it did before.
* Verify that the removed code is not in use.
* Verify that all tests pass.
